src/resources/experimental/transactions-resource.js

```diff
@@ -0,0 +1,7 @@
+export default function TransactionsResource({apiHandler}) {
+    return {
+        async reschedule({id, data}) {
```

Comment:
Since `id` is inserted into the path, it should default to an empty string: `{id = '', data}`.